lib/releaselib.sh

```diff
@@ -631,13 +631,30 @@ release::gcs::locally_stage_release_artifacts() {
       || return 1
     release::gcs::stage_and_hash $gcs_stage $gci_path/configure.sh extra/gce \
       || return 1
+
     # shutdown.sh was introduced starting from v1.11 to make Preemptible COS nodes
     # on GCP not reboot immediately when terminated. Avoid including it in the release
     # bundle if it is not found (for backwards compatibility).
     if [[ -f $gci_path/shutdown.sh ]]; then
       release::gcs::stage_and_hash $gcs_stage $gci_path/shutdown.sh extra/gce \
        || return 1
     fi
+
+    # Having the Windows startup scripts from the GCE cluster deploy hosted with
+    # the release is useful for GKE.
+    win1803_local_path=$gce_path/win1803
+    win1803_gcs_path=extra/gce/win1803
+    if [[ -f $win1803_local_path ]]; then
+      release::gcs::stage_and_hash $gcs_stage $win1803_local_path/configure.ps1 $win1803_gcs_path \
+        || return 1
+      release::gcs::stage_and_hash $gcs_stage $win1803_local_path/common.psm1 $win1803_gcs_path \
+        || return 1
+      release::gcs::stage_and_hash $gcs_stage $win1803_local_path/k8s-node-setup.psm1 $win1803_gcs_path \
+        || return 1
+      release::gcs::stage_and_hash $gcs_stage $win1803_local_path/user-profile.psm1 $win1803_gcs_path \
```

Comment:
This is only for installing sshd. We probably don'e need to package it for official release. 
/cc @pjh

Reply:
Right, I realized last week that I accidentally included `user-profile.psm1` in the PR even though it's unused at the moment. I think we still plan to enable SSH for our k/k test clusters soonish, but we don't need the file for official release.

If it would make things easier we could put the "non release" scripts in a separate subdir, or remove them from the main tree completely.

Reply:
Making it clear, even just putting "test" or something like that in the filename, would be useful.